config-model/src/main/java/io/strimzi/kafka/config/model/ConfigModels.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.config.model;
+
+import java.util.Map;
+
+/**
+ * Models all the broker configuration parameters for a given version of the Kafka broker.
```

Comment:
talking about "broker" here let me think, following the previous question, that this config model will be just for broker configuration, or?

Reply:
Right now it's just the broker configs.